backend/strawberry_forms/tests/test_mutations.py

```diff
@@ -1,73 +0,0 @@
-import strawberry
```

Comment:
I didn't manage to get the mutation to work, if someone has any idea happy to hear it, otherwise, I would just delete this code and move on removing the rest of strawberry_forms